ui/app/adapters/cluster.js

```diff
@@ -169,7 +169,7 @@ export default ApplicationAdapter.extend({
   urlFor(endpoint) {
     if (!ENDPOINTS.includes(endpoint)) {
       throw new Error(
-        `Calls to a ${endpoint} endpoint are not currently allowed in the vault cluster adapater`
+        `Calls to a ${endpoint} endpoint are not currently allowed in the vault cluster adapter`
```

Comment:
Nice catch! Should we also capitalize Vault while here? 
```suggestion
        `Calls to a ${endpoint} endpoint are not currently allowed in the Vault cluster adapter`
```

Reply:
This is more of a dev message, so it should never show to users. That being said, I'm indifferent on updating it